src/components/tutorial-progress-icon/tutorial-progress-icon.module.css

```diff
@@ -0,0 +1,21 @@
+.progressIcon {
+	&.status_in_progress {
+		--color: var(--token-color-palette-blue-200);
```

Comment:
Have confirmed with design that we want to use `--token-color-palette-blue-200`.

Note: still need to implement the subtle colour variation on hover here. I'm leaving this for a separate task as I think it'll be enough to warrant a separate PR, since we'll need to pass in `active` and `hover` states somehow (perhaps through CSS custom properties? Or through props?) 🎟 [[progress-icon] add hover and active color states](https://app.asana.com/0/1202097197789424/1202964504180132/f)